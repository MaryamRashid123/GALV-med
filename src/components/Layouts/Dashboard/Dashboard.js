/*
  Application Dashboard
*/

import React from 'react';
import { Row, Col } from 'antd'

function Dashboard({ title }) {
  
  return (
    <div className="secondary-header">
      <Row gutter={[16, 8]} className="align-items-center">
        {/* Title */}
        <Col span="10">
          <h1>{ title }</h1>
        </Col>
      </Row>
    </div>
  );
}

export default Dashboard;
