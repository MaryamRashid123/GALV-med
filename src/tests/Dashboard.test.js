import React from 'react';
import { shallow } from 'enzyme';
//import Title from './Title';
import Dashboard from '../components/Layouts/Dashboard/Dashboard'

let wrapped = shallow(<Dashboard title="Dashboard"/>);
describe('dashboard', () => {
  it('should render the Component correctly', () => {   
    expect(wrapped).toMatchSnapshot();
  });
  
});