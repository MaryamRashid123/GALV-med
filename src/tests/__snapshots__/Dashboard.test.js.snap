// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dashboard should render the Component correctly 1`] = `
<div
  className="secondary-header"
>
  <Row
    className="align-items-center"
    gutter={
      Array [
        16,
        8,
      ]
    }
  >
    <Col
      span="10"
    >
      <h1>
        Dashboard
      </h1>
    </Col>
  </Row>
</div>
`;
